openshift-hack/images/os/install.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -xeou pipefail
+
+yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
+curl http://base-4-10-rhel8.ocp.svc > /etc/yum.repos.d/rhel8.repo
+curl http://base-4-10-rhel8-aarch64.ocp.svc > /etc/yum.repos.d/rhel8-aarch64.repo
```

Comment:
In theory, it should be possible to drop this since all the repos that RHCOS is built from should come from `base-4-10-rhel8.ocp.svc`

But the jobs seem to be producing different results compared to me running a pod on app.ci or build01 and curling the same hostname...